publisher:
    boxes:
        topic: /detector/bounding_boxes
        queue_size: 10
        latch: false

    image:
        topic: /detector/bounding_boxes/image
        queue_size: 1
        latch: false

subscriber:
    image:
        topic: /camera/color/image_raw
        queue_size: 1
